tests/test_models.py

```diff
@@ -6,22 +6,77 @@
 
 from __future__ import absolute_import, unicode_literals
 
+import unittest
 
-class TestEnterpriseCustomer(object):
+import freezegun
+import datetime
+import ddt
+from pytest import mark
+
+from enterprise.models import EnterpriseCustomer
+from tests.factories import EnterpriseCustomerHistoryFactory, EnterpriseCustomerFactory, EnterpriseCustomerMemberFactory
+
+
+@mark.django_db
+class TestEnterpriseCustomerManager(unittest.TestCase):
+    def tearDown(self):
+        EnterpriseCustomer.objects.all().delete()
+
+    def test_get_queryset_returns_only_active(self):
+        customer1 = EnterpriseCustomerFactory(active=True)
+        customer2 = EnterpriseCustomerFactory(name="Customer 2", active=True)
+        inactive_customer = EnterpriseCustomerFactory(name="Inactive", active=False)
+
+        all_customers = EnterpriseCustomer.objects.all()
+        self.assertTrue(all(customer.active for customer in all_customers))
+        self.assertIn(customer1, all_customers)
+        self.assertIn(customer2, all_customers)
+        self.assertNotIn(inactive_customer, all_customers)
+
+
+@mark.django_db
+@ddt.ddt
+class TestEnterpriseCustomer(unittest.TestCase):
     """
     Tests of the EnterpriseCustomer model.
     """
+    @ddt.data(
+        str, repr
+    )
+    def test_string_conversion(self, method):
+        customer = EnterpriseCustomerFactory(id=150, name="QWERTY")
+        expected_to_str = "<EnterpriseCustomer 150: QWERTY>"
+        self.assertEqual(method(customer), expected_to_str)
 
```

Comment:
nit: Consider adding a test that makes sure a new `EntepriseCustomerHistory` instance is created every time a new `EnterpriseCustomer` model is created or updated.